wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,207 @@
+| White | Common time |
+| Green | Current timestamp / [Judgement](/wiki/Gameplay/Judgement) line |
 
-#### Hitsounds
-
-| Button (Keyboard Shortcut) | Usage | Description |
-| :-- | :-- | :-- |
-| Whistle (`W`) | ("Select" on a Hit note) **Left-Click:** Use whistle sound on this note | Add a whistle sound addition to the selection. |
-| Finish (`E`) | ("Select" on a Hit note) **Left-Click:** Use finish sound on this note | Add a finish (cymbal) sound addition to the selection. |
-| Clap (`R`) | ("Select" on a Hit note) **Left-Click:** Use clap sound on this note | Add a clap sound addition to the selection. |
-
-#### Assist Tools
-
-| Button (Keyboard Shortcut) | Usage | Description |
-| :-- | :-- | :-- |
-| Grid Snap (`T`) | **`Shift` (Hold):** Temporary toggle. **`Ctrl` (Hold):** Temporary disable angle snapping. | Hit notes will be snapped to the grid by default. |
-| Distance Snap (`Y`) | **`Alt` (Hold):** Temporary toggle. Switch Beat Snap Divisor to Distance Snap **`Alt` + Mousewheel:** Adjust Distance Snap multiplier. | Distance between consecutive hit notes will be snapped based on their rhythmical difference. Best use while the timeline is paused. |
-| Lock Notes (`L`) | ("Select" on a Hit note) **Left-Click:** Lock this note | Lock selected note to current position and timestamp. |
-
-For **keysound** (adding SE on hitnotes), select the note and **`Alt` + Left Click** on the selected note (a pop-up window will be shown with sound samples inside your beatmap folder). Refer to "Sampling" section at end of page.
-
-### Bottom (Song Timeline)
-
-![Song Timeline](/wiki/shared/BE_STL.jpg "Song Timeline")
-
-At **bottom-left**, it shows **timestamp in milliseconds (ms)** and **song duration %**.
-
-At **bottom-centre**, it shows the **timeline with markings and the compulsory music player buttons.** For "Test" button, your beatmap will be saved to test the beatmap starting on current timestamp.
-
-#### Bookmarking command
-
-| Shortcut | Description |
+| Note colour | Description |
 | :-- | :-- |
-| `Ctrl` + `B` | Add bookmark at current location. |
-| `Ctrl` + `Shift` + `B` | Remove bookmark at current location. |
-| `Ctrl` + `Right` Arrow | Next bookmark. |
-| `Ctrl` + `Left` Arrow | Previous bookmark. |
-
-At **bottom-right**, you can adjust the **Playback Rate** by **per quarter value.**
+| Blue | Selected note(s) |
+| White/Pink/Yellow | Unselected note colours |
 
 ### Sampling
 
-![Sample pop-up window (Windows 8's ver.)](img/Sample_omc.jpg "Sample pop-up window (Windows 8's ver.)")
-
-**Sampling** is a process of **adding sound to the note.** To begin, select the note and **`Alt` + Left Click** on the selected note (a pop-up window will be shown with sound samples inside your beatmap folder).
+**Sampling** is the process of adding sound to the note. To begin, select the note and `Alt` **+** `Left click` on the selected note and a pop-up window will be shown with sound samples inside the beatmap folder.
```

Comment:
also, is it really called sampling? always though it's keysounding, requesting osu!mania mappers' support here
```suggestion
**Sampling** is the process of adding sound to the note. To add a sample, click a note while holding `Alt` to open a pop-up window with a list of sound samples inside the beatmap folder.
```